lib/matplotlib/animation.py

```diff
@@ -1299,7 +1299,20 @@ def to_html5_video(self, embed_limit=None):
             return 'Video too large to embed.'
 
     def to_jshtml(self, fps=None, embed_frames=True, default_mode=None):
-        """Generate HTML representation of the animation"""
+        """
+        Generate HTML representation of the animation.
+
```

Comment:
One major usecase is to embed the animation into a jupyter notebook. Maybe that should be mentioned. See here for an example: https://github.com/timhoffm/euroscipy2019-using-matplotlib/blob/master/06_Animations.ipynb